tests/test_topology.py

```diff
@@ -9,26 +9,26 @@ class TestSystemTopology:
         Test :py:class:`system.SystemTopology`.
     """
 
-    def test_parse(self):
+    def test_parse_single_intel_core_i7_4790(self):
```

Comment:
You could create a fake `xml` output for the `Raptor Lake` CPU, using https://www.open-mpi.org/projects/hwloc/lstopo/images/RaptorLake-hybrid.v2.10.png.